shex/carePathwayShape.shex

```diff
@@ -1,21 +1,30 @@
-PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
-PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX : <http://purl.org/ejp-rd/cde/v030/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
 PREFIX sio: <http://semanticscience.org/resource/>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
-:personShape IRI {
-  a [sio:SIO_000498];
-  sio:SIO_000228 @:roleShape
+:identifierShape IRI {
+    a [sio:SIO_000115];
+    sio:SIO_000020 @:carePathwayRoleShape;
+    sio:SIO_000300 xsd:string
 }
 
-:roleShape IRI {
-  a [obo:OBI_0000093];
-  sio:SIO_000356 @:processShape
+:personShape IRI { 
+    a [sio:SIO_000498] ;
+    sio:SIO_000228 @:carePathwayRoleShape
 }
 
-:processShape IRI {
-  a [sio:SIO_000006];
-  a [obo:NCIT_C142427];
-  a [obo:NCIT_C159705];
-  sio:SIO_000680 xsd:dateTime
+:carePathwayRoleShape IRI {
+    a [obo:OBI_0000093];
+    a [sio:SIO_000016];
+    rdfs:label xsd:string;
```

Comment:
I understand it would be very helpful to have human-readable annotations here. If it is written like this, it would be **MANDATARY** for any Turtle. If that is what you want, it is correct, otherwise, set it as optional. 
Personally speaking, I like it because it increases the data quality in terms of interoperability :D

Reply:
Exactly the same as the `optional` suggestion from Rajaram.